src/main/java/de/qabel/desktop/ui/AbstractController.java

```diff
@@ -150,4 +152,29 @@ public Contact gsonContactToContact(GsonContact gc, Identity i) throws URISyntax
 		}
 		return collection;
 	}
+	protected String calculateTimeString(DropMessage dropMessage) {
+
+		Long messageDate = dropMessage.getCreationDate().getTime();
+
+		java.util.Date date= new java.util.Date();
+		Timestamp currentTimestamp = new Timestamp(date.getTime());
+
+		long diff = currentTimestamp.getTime() - messageDate;
+
+		int minutes = (int) ((diff / (1000*60)));
+		int hours   = (int) ((diff / (1000*60*60)));
+
+		String text;
+		if (minutes < 10) {
+			text = "moments ago";
+		} else if (minutes < 60) {
+			text = String.valueOf(minutes + " minutes ago");
```

Comment:
`String.valueOf(minutes + " minutes ago");` should be equal to `minutes + " minutes ago"` because it is auto converted to a String on concatenation. Otherwise, only `String.valueOf(minutes) + " minutes ago" would make sense